gtsam/geometry/OrientedPlane3.h

```diff
@@ -114,6 +114,10 @@ class GTSAM_EXPORT OrientedPlane3 {
    */
   Vector3 error(const OrientedPlane3& plane) const;
 
+  static Vector3 Error(const OrientedPlane3& plane1, const OrientedPlane3& plane2) {
```

Comment:
I don't think we should add this. Instead, add 2 OptionalJacobian arguments to method above (H1 for this, H2 for plane)